pkg/auth/providers/azure/azure_provider.go

```diff
@@ -50,12 +51,16 @@ func Configure(
 	tokenMGR *tokens.Manager,
 ) common.AuthProvider {
 
+	// Setup the groupCache, ignore the error, it will only error on a non-positive number
+	c, _ := lru.New(10000)
```

Comment:
Should this be a settings?  Seems like we always hit this with LRU that when you end up with N+1 possibilities we end up trashing the cache.

Reply:
Good call. Updated